example/ck_tile/36_copy/test_copy.hpp

```diff
@@ -150,40 +143,44 @@ struct TileCopy
             __builtin_amdgcn_readfirstlane(integer_divide_ceil(N, S::Block_N));
         auto my_id = get_warp_id();
 
-        auto DramTileDist   = x_block_window.get_tile_distribution();
-        using dram_reg_tile = decltype(make_static_distributed_tensor<XDataType>(DramTileDist));
-
         for(int iN = __builtin_amdgcn_readfirstlane(0); iN < num_n_tile_iteration; ++iN)
         {
-            dram_reg_tile dram_tile;
-
             if(my_id == warp_id)
             {
                 if constexpr(AsyncCopy)
                 {
-                    async_load_tile(x_block_lds_window_no_dist, x_block_window);
+                    async_load_tile(x_block_lds_write_window, x_block_window);
+
+                    // Wait all asyncload insts complete.
+                    __builtin_amdgcn_s_waitcnt(3952);
```

Comment:
should we consider substituting 3952 with something like `kExportCnt << 4 | kLDSCnt << 8`? And also having a constexpr helper function somewhere?